src/Markdown.MAML/Renderer/MamlRenderer.cs

```diff
@@ -75,331 +31,161 @@ public string MamlModelToString(IEnumerable<MamlCommand> mamlCommands)
         /// <returns></returns>
         public string MamlModelToString(IEnumerable<MamlCommand> mamlCommands, bool skipPreambula)
         {
-            _stringBuilder.Clear();
-            if (!skipPreambula)
-            {
-                _stringBuilder.AppendLine(XML_PREAMBULA);  
-            }
-
-            AddCommands(mamlCommands);
+            var document = new XDocument(
+                new XDeclaration("1.0", "utf-8", null),
```

Comment:
I tried to generate platyPS help with you changes and got

```
<?xml version="1.0" encoding="utf-16"?>
```

I wonder why it didn't pick up encoding from this line?